### DISCUSSION TITLE
docker中的fastdfs如何扩充内存

### DISCUSSION THREAD

**Discussion by Adou02** [community/community#65878] [Programming Help]
### Body

我现在的是需求是, 爬取很多PDF或者word文档, 但是要解决跨域问题就必须将文件下载到本地, 我通过docker安装fastdfs容器去存储的话现在提示我内存不足,我在网上看了一下扩充的方法, 但是在我的容器里找不到tracker.conf之类的配置文件

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by arafajs** [comment]
The tracker.conf file is usually located in the /etc/fdfs directory of the fastdfs container. However, if you cannot find it, you can try the following steps to expand the memory of the container:

1. Stop the fastdfs container:

```
docker stop fastdfs
```

2. Increase the memory limit of the container:

```
docker update --memory=4096m fastdfs
```

3. Start the fastdfs container again:

```
docker start fastdfs
```

Once the container has been restarted, the tracker.conf file should be available in the /etc/fdfs directory. You can then edit this file to increase the memory allocation for the fastdfs server.

Here is an example of how to increase the memory allocation for the fastdfs server:

```
tracker_memory = 4096
```

This will allocate 4096MB of memory to the fastdfs server. You can adjust this value as needed.

Once you have edited the tracker.conf file, you need to restart the fastdfs server for the changes to take effect. You can do this by running the following command:

```
docker restart fastdfs
```

After the fastdfs server has been restarted, you should be able to store more files in the container without running out of memory.
